src/components/Navigator.vue

```diff
@@ -9,7 +9,10 @@
 -->
 
 <template>
-  <nav class="navigator">
+  <nav
+    v-if="displayNavigator"
```

Comment:
I am worried this will cause allot of unnecessary re-rendering. Wont applying an `aria-hidden` in the DocumentationTopic be enough?

Reply:
Applying only `aria-hidden` will prevent VO navigation but not keyboard navigation. An alternative is applying `display: none` which will prevent keyboard navigation too but I thought it would be better to just not rendering it at all.

Reply:
This would also I think remove any transition animations we have, when opening/closing the nav on mobile, so we need to think of a better way.

Reply:
Updated using v-show: https://github.com/apple/swift-docc-render/pull/296/commits/a70e5c79fa65edb0d93ff6323456f6529d15b427

Reply:
I tested on mobile and the animation seemed fine.